src/material/autocomplete/autocomplete.spec.ts

```diff
@@ -2119,6 +2119,39 @@ describe('MatAutocomplete', () => {
     }));
   });
 
+  describe('with panel classes in the default options', () => {
```

Comment:
Can you copy these tests into the MDC autocomplete too?

Reply:
Oh yeah for sure, good call